test/e2e/tests/bridge/bridge-test-utils.ts

```diff
@@ -87,6 +87,7 @@ export async function bridgeTransaction(
   quote: BridgeQuote,
   transactionsCount: number,
   expectedWalletBalance?: string,
+  repeatedTransactionsCount?: number,
```

Comment:
this is not ideal. Instead of having 1 function `bridgeTransaction` which does everything (send and validate) we should instead build POM flows for these  ((one flow for bridge and one flow for validate tx activity). However, I leave this out of scope of this PR, as this PR only intended to fix the flakiness of the `checkTxAction`method`